source/includes/usage-examples/connect-code-examples.cpp

```diff
@@ -0,0 +1,103 @@
+#include <mongocxx/instance.hpp>
+#include <mongocxx/client.hpp>
+#include <mongocxx/uri.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/exception/exception.hpp>
+
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_document;
+
+int main()
+{
+    mongocxx::instance instance;
+
+    {
+        // Retrieves one document that matches a query filter
+        // start-local
+        mongocxx::uri uri("mongodb://localhost:27017/");
+        mongocxx::client client(uri);
+        // end-local
+    }
+
+    {
+        // start-atlas
+        mongocxx::uri uri("<Atlas connection string>");
+        mongocxx::client client(uri);
+        // end-atlas
+    }
+
+    {
+        // Retrieves one document that matches a query filter
+        // start-replica-set
+        mongocxx::uri uri("mongodb://<replica set member>:<port>/?replicaSet=<replica set name>");
+        mongocxx::client client(uri);
+        // end-replica-set
+    }
+
+    {
+        // start-enable-tls
+        mongocxx::uri uri("mongodb://<hostname>:<port>/?tls=true");
+        mongocxx::client client(uri);
+        // end-enable-tls
+    }
+
+    {
+        // start-disable-ocsp
+        mongocxx::uri uri("mongodb://<hostname>:<port>/?tls=true&tlsDisableOCSPEndpointCheck=true");
+        mongocxx::client client(uri);
+        // end-disable-ocsp
+    }
+
+    {
+        // start-crl
+        mongocxx::options::client client_options;
+        mongocxx::options::tls tls_options;
+
+        tls_options.crl_file("/path/to/file.pem");
```

Comment:
S: Use our placeholder convention here to make this stand out more as something that needs to be replaced
```suggestion
        tls_options.crl_file("<path to your CRL file>");
```